### ISSUE TITLE
Create Tab-Section-Field generator

### ISSUE THREAD

**Issue by ivankristianto** [sofyansitorus/WordPress-Yet-Easy-Settings#1]
It would be very good to provide automation script to generate tab-section-fields :)

**Comment by sofyansitorus** [comment]
Yes, that will be nice features to have generator for the settings tabs and fields using drag and drop interface.
